src/shipit_workflow/shipit_workflow/release.py

```diff
@@ -10,16 +10,35 @@
 FINAL_RELEASE_REGEX = '^\d+\.\d+$'
 
 
+VERSION_REGEX = re.compile(
+    '^'
+    '(?P<major_minor>[0-9]+\.[0-9]+)'  # Major and minor version
+    '(?:'  # Patch or beta version is optional
```

Comment:
Can you use raw strings (`r'...'`) instead. It'd be just safer in the future in case we want to change the regexp.